go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/treeverse/lakefs
 
-go 1.14
+go 1.15
```

Comment:
Not sure about this.  I agree that we _should_ test also with bleading-edge Go.  But it is impractical to _require_ this version in order to compile.  I would feel better about continuing to use 1.14 for a while, and commit to support 1.15 and 1.14 by running GitHub testing actions in a matrix.
WDYT?

Reply:
No commit to supporting a specific version - if we will have issues with go 1.15 we can rollback the change.
Prefer to use what any contributor will get when he/she does 'brew install go'.
Addressing the current Go version in PR #674.
Can defer the upgrade but I don't have a good timeline on when it is the right time to switch, can go with always one version back - or only after X maintenance releases of the Go toolchain.